crates/vm/levm/tests/tests.rs

```diff
@@ -4643,3 +4646,50 @@ fn ecpairing_test() {
         (ECPAIRING_BASE_COST + ECPAIRING_GROUP_COST * 2)
     );
 }
+
+#[test]
+fn blake2f_test() {
```

Comment:
Maxi, what do you think about adding a source for these tests? Just in the case in the future it's easier to find them

Something like: 
```suggestion
#[test]
// Source: https://eips.ethereum.org/EIPS/eip-152#test-vector-1
fn blake2f_test() {
```

Reply:
Done in [Add source of first test](https://github.com/lambdaclass/ethrex/pull/1563/commits/42fa3c8eb596fd21cd8c1fc6602c25cd4de9ba59)